openpitrix/values.yaml

```diff
@@ -30,7 +28,17 @@ memory_requests: 100
 cpu_limits: 500
 memory_limits: 500
 
-mysql_password: password
+etcd:
```

Comment:
There are some new env in dashboard, have a look at docker-compose.yaml in openpitrix.